docs/apis/hydrateStore.md

```diff
@@ -0,0 +1,53 @@
+### `hydrateStore( resource: object ): Promise`
+
+"Hydration" is a term for filling an object with data. In this case, we are
+"hydrating" the store on startup of an application using the `hydrateStore`
+action.
+
+Dispatch function returned from `hydrateStore` to add a JSON-API payload to the
+store.
+
+### Example
+
+```js
+import { createStore, combineReducers } from 'redux';
+import { reducer as api, hydrateStore } from 'react-redux';
+
+const store = createStore(
+  combineReducers(api)
+);
+
+const jsonBlob = {
+  "data": [{
+    "type": "tasks",
+    "id": "1",
+    "attributes": {
+      "task": "Deliver presents"
+    },
+    "relationships": {
+      "author": {
+        "data": { "type": "users", "id": "1" }
+      }
+    }
+  }],
+  "included": [{
+    "type": "users",
+    "id": "1",
+    "attributes": {
+      "username": "santa"
+    }
+  }]
+};
+
+store.dispatch(hydrateStore(jsonBlob));
```

Comment:
I think what may be confusing is that this is not the regular use pattern, since `dispatch` is normally passed as prop from `react-redux`. I'm good with this though.